### ISSUE TITLE
PSR12 standard reports errors for multi-line FOR definitions

### ISSUE THREAD

**Issue by andre-exads** [squizlabs/PHP_CodeSniffer#2763]
Current PSR-12 configuration on CodeSniffer treat multi-line conditions in a for loop as incorrect, but according to https://www.php-fig.org/psr/psr-12/#54-for it is allowed. Example bellow.
```php
for (
    $i=0;
    $i<5;
    $i++
) {
    // body here
}
```

**Comment by jrfnl** [comment]
@andre-exads Could you share the ruleset configuration you are using ? Or at the very least, please check that your ruleset does not include both `PSR2` as well as `PSR12`.
The `PSR12` ruleset includes all relevant rules from `PSR2` automatically and having both rulesets active causes conflicts as the standards are different.

**Comment by andre-exads** [comment]
I'm not using phpcs file of any king in my project (when I tried it was an even bigger mess) I only have the following setting on VScode settings.json:
```json
"phpcs.standard": "PSR12",
```

**Comment by gsherwood** [comment]
I've committed a fix for this, which will be in 3.5.4. The solution required a new `ignoreNewlines` property for the `Squiz.ControlStructures.ForLoopDeclaration` sniff, which I've documented here: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties#squizcontrolstructuresforloopdeclaration

Thanks for reporting this.
